lib/assets/javascripts/turbograft/page.coffee

```diff
@@ -19,9 +19,12 @@ Page.refresh = (options = {}, callback) ->
   if options.response
     onlyKeys   = options.onlyKeys   || []
     exceptKeys = options.exceptKeys || []
-    Turbolinks.loadPage null, options.response, true, callback, onlyKeys, exceptKeys
+    Turbolinks.loadPage null, options.response, true, true, callback, onlyKeys, exceptKeys
```

Comment:
this method signature is looking sooo messy (not your fault :P)
Would you be able to refactor it to accept an options hash as part of this PR?